.cicd/base-images-pinned.yml

```diff
@@ -48,7 +48,7 @@ steps:
     agents: "queue=mac-anka-node-fleet"
     timeout: 180
     skip: ${SKIP_MACOS_10_14}${SKIP_MAC}
-    
+
   - label: ":ubuntu: Ubuntu 16.04 - Base Image Pinned"
```

Comment:
I noticed there is a ubuntu 16.04 pinned image build but not unpinned. This seems like a mistake?